core/chains/evm/config/v2/config.go

```diff
@@ -464,7 +464,7 @@ type GasEstimator struct {
 	BumpMin       *assets.Wei
 	BumpPercent   *uint16
 	BumpThreshold *uint32
-	BumpTxDepth   *uint16
+	BumpTxDepth   *uint32 `toml:",omitempty"`
```

Comment:
What is the effect of using `omitempty` here?

Reply:
Good catch - I needed it for a previous way of implementing this, but I can remove it since I don't have unit test coverage on the annotation anymore.